lib/WUI/netdev.c

```diff
@@ -430,3 +321,20 @@ const char *netdev_get_hostname(uint32_t active_id) {
         return "";
     }
 }
+
+void netdev_join_ap(uint32_t active_id) {
```

Comment:
The `active_id` doesn't particularly make sense in here. Only ESP can join AP and it doesn't have to be active for that.

Reply:
Sure, just a copy paste. Parameter removed.